RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -779,9 +779,6 @@ def verifyRbfwVersion(self):
             self.versioncontrol_error('conflict_max', self.sMaxVersion, sCurrentVersion)
             bVersionCheck = False
 
-        if bVersionCheck:
```

Comment:
Now `bVersionCheck` is set, but not used. This makes no sense.
Nevertheless: You removed a mandatory message. Why? A result of an executed version check has to be logged.